app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -25,17 +28,36 @@ def description
   end
 
   def send_report!
-    HTTParty.post("#{ArchiveConfig.NEW_BUGS_SITE}#{project_path}",
-                  body: "&xml=#{URI.encode_www_form_component(xml.to_str)}")
+    zoho_resource_client.create_ticket(
+      ticket_attributes: report_attributes,
+    )
+  end
+
+  def report_attributes
+    {
+      "email" => email,
+      "contactId" => zoho_contact_id,
+      "cf" => {
+        "Language" => language.presence || "English",
```

Comment:
Do we want to hard code English here or get the language from `Locale.default.language`?

Reply:
I'm going with `Language.default.name` since we use Language objects for the dropdowns.